src/python/install.sh

```diff
@@ -26,7 +26,7 @@ CONFIGURE_JUPYTERLAB_ALLOW_ORIGIN="${CONFIGUREJUPYTERLABALLOWORIGIN:-""}"
 # alongside PYTHON_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv")
```

Comment:
Can you bump the patch version in here 👇 ? Thanks
https://github.com/devcontainers/features/blob/3f2738f0217e9f6e5dc441fa105f2dca7f1e00d0/src/python/devcontainer-feature.json#L3

Reply:
Done!